# An array of PointStamped points

read_msgs/Frontier[] frontiers